lib/cmd/fh3/appdata.js

```diff
@@ -0,0 +1,180 @@
+module.exports = appdata;
+
+appdata.desc = "Export FeedHenry MBaaS Apps";
+appdata.usage = "fhc appdata [cmd]"
+    + "\nfhc appdata export --app=<app> --env=<environment>"
+    + "\nfhc appdata list"
+    + "\nfhc appdata download --job=<job-id>"
+    + "\nwhere <job-id> is the auto generated job id"
+    + "\nwhere <app> is the app id"
+    + "\nwhere <environment> is the environment id";
+
+var common = require("../../common");
+var fhreq = require("../../utils/request");
+var fhc = require("../../fhc");
+var ini = require('../../utils/ini');
+var _ = require('underscore');
+var templates = require('../common/templates.js');
+var readline = require('readline');
+
+var cmdRef = {
+  'list': listAppData,
+  'export': exportAppData,
+  'download': downloadAppData,
+  'unknown': unknown
+};
+
+function appdata(argv, cb) {
+  var args = argv._;
+  if (args.length === 0) {
+    return unknown("", cb);
+  }
+
+  var action = args[0];
+  var fn = cmdRef[action];
+
+  if (!fn) {
+    return unknown("Invalid service action " + action, cb);
+  }
+
+  return fn(argv, cb);
+}
+
+function listAppData(args, cb) {
+  var app = args.app;
```

Comment:
Maybe call it `addId` to make clear that it's about the app id (and not it's label)